package.json

```diff
@@ -138,12 +138,12 @@
         ]
       },
       {
-        "permissionName": "ui-organizations.all",
+        "permissionName": "ui-organizations.module.all",
         "displayName": "Organizations: all permissions for Create, View, Edit and Delete organizations",
         "visible": true,
         "description": "",
         "subPermissions": [
-          "organizations-storage.all"
+          "ui-organizations.module.all"
```

Comment:
isn't it a circular dependency? You have sub perm `ui-organizations.module.all` inside perm `ui-organizations.module.all`

Reply:
maybe you meant https://github.com/folio-org/mod-organizations-storage/blob/master/descriptors/ModuleDescriptor-template.json#L574

Reply:
My fault. Fixed